generator/source/faq/installation.rst

```diff
@@ -279,7 +270,7 @@ It is just a warning and you can ignore it. Verify the Sawtooth repository was a
 
 I get this error starting Sawtooth on Docker: ``No response from ... beginning heartbeat pings``
 ------------------------------------------------------------------------------------------------
-This means there is a problem with the genesis node and peer nodes connecting.
+It means that there has been no message exchange between the validators for 10 seconds. This could be a problem with Raft or PBFT, but not with other consensus engines.
```

Comment:
Is this accurate? If so, still needs rewording to be stated with less opinion.

Reply:
I'm also not if sure this is accurate. Additionally, the wording is problematic because it seems to imply: 1) that Raft or PBFT is *causing* the problem, which is definitely not true, and 2) if you use any other consensus engine (even one that is not implemented yet), you would never see this message.

Reply:
@vaporos  I removed the Raft/PBFT sentence.  Please re-review